### PR TITLE
[SPARK-12080] [SparkCore] Kryo - Support multiple user registrators

### PR DESCRIPTION
**Pull request by RotemShaul**

### CODE REVIEW

**Comment by JoshRosen** [comment]
How are users supposed to discover this feature? Would you mind updating the docs in `docs/configuration.md` to make it clear that this accepts a comma-separated list of classes?

**Comment by Botnaim** [comment]
You're right, I've missed that important bit.
I've added it to the PR.

**Comment by JoshRosen** [comment]
Jenkins, this is ok to test.

**Comment by Botnaim** [comment]
I updated the code to fit the 100-char per line limit.

**Comment by Botnaim** [comment]
Seems to fail for different reasons, any idea why ?

**Comment by vanzin** [comment]
retest this please

File: docs/configuration.md [hunk 1]
```diff
@@ -647,10 +647,10 @@ Apart from these, the following properties are also available, and may be useful
   <td><code>spark.kryo.registrator</code></td>
   <td>(none)</td>
   <td>
-    If you use Kryo serialization, set this class to register your custom classes with Kryo. This
+    If you use Kryo serialization, give a comma-separated list of classes that register your custom classes with Kryo. This
     property is useful if you need to register your classes in a custom way, e.g. to specify a custom
     field serializer. Otherwise <code>spark.kryo.classesToRegister</code> is simpler. It should be
-    set to a class that extends
+    set to classes that extends
```
Comment (vanzin) [inline docs/configuration.md]: "classes that extend"

**Comment by Botnaim** [comment]
@vanzin  I updated the documentation according to your comment.

The tests seemed to pass now.

**Comment by vanzin** [comment]
Merging to master.

**Comment by Botnaim** [comment]
What's the status of merging ?

**Comment by vanzin** [comment]
This is in master already. No plan to put this in 1.6 at the moment since it's in the RC phase and this is not a regression.

**Pull request closed without merging**